gh-120284: Enhance asyncio.Runner's run() method to accept more object types (#120566)
python/cpython: Doc/library/asyncio-runner.rst

.. function:: run(coro, *, debug=None, loop_factory=None)

>    Execute code in an asyncio event loop and return the result.

Reviewer: ```suggestion
   Execute the :term:`awaitable` *coro* and return the result.
```
